auslib/migrate/versions/026_rename_system_capabilities.py

```diff
@@ -3,6 +3,8 @@ def upgrade(migrate_engine):
     # Migration disabled prior to code going to production. This was replaced by
     # a two part migration that added instructionSet in one migration, and
     # removed systemRequirements in another.
+
```

Comment:
This is a bit strange too. Maybe because the # below are on the first column vs. indented?

Reply:
Yeah, seems likely.